content/en/blog/_posts/XXXX-XX-XX-hpa-configurable-tolerance.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: "Horizontal Autoscaling Configurable Tolerance"
+slug: hpa-configurable-tolerance
+date: XXXX-XX-XX
+author: "Jean-Marc François"
+---
+
+This post describes _Configurable Tolerance for Pod Horizontal Autoscaling_,
+a new alpha feature first available in Kubernetes 1.33.
+
+## What is it?
+
+Horizontal Pod Autoscaling (HPA) is a well-known Kubernetes feature that
+allows your workload to automatically resize by adding or removing replicas
+based on resource utilization.
+
+To decide how many replicas a workload requires, users configure their HPA
+with a metric (e.g. CPU utilization) and an expected value for this metric (e.g.
+80%). The HPA updates the number of replica based on the ratio between the
+current and desired metric value. (For example, if there are currently 100
+replicas, the CPU utilization is 88%, and the desired utilization is 80%, the
+HPA will ask for `100 * (88/80)` replicas).
```

Comment:
We prefer rendered formulae; see https://kubernetes.io/docs/contribute/style/style-guide/#formulae-and-equations

You can also use `×` rather than `*` here, but a formula is what we recommend.